README.md

```diff
@@ -162,6 +162,21 @@ Windows has support for [unix domain socket](https://devblogs.microsoft.com/comm
 
 On MacOS Mojave, setting more than `2048` bytes for `StatsdMaxUnixDomainSocketPacketSize` is experimental.
 
+## Client side aggregation
+
+By default, metrics for basic types (gauges, counts, sets) are aggregated before they are sent. For example, instead of sending 3 times `my_metric:10|c|#tag1:value`, DogStatsD client sends `my_metric:30|c|#tag1:value` once. 
+For more technical details about how client-side aggregation works see #134.
```

Comment:
What is `#131` in this context?

Reply:
#134 is the link to the PR that introduced client side aggregation. 
Note: Github will create a full link from `#134`